ocs_ci/ocs/registry.py

```diff
@@ -333,3 +333,26 @@ def check_image_exists_in_registry(image_url):
         return_value = True
         logger.info("Image exists in Registry")
     return return_value
+
+
+def remove_ocp_registry_from_ocs(platform):
+    """
+    Function removes OCS registry from OCP cluster
+
+    Args:
+        platform (str): the platform the cluster deployed on
+
+    """
+    image_registry_obj = ocp.OCP(
+        kind=constants.CONFIG, namespace=constants.OPENSHIFT_IMAGE_REGISTRY_NAMESPACE
+    )
+    param_cmd = ''
+    if platform == 'AWS':
```

Comment:
better to save `AWS` and `VMWARE` in constants.py

Reply:
they exist there but is lower case 
AWS_PLATFORM = 'aws'
VSPHERE_PLATFORM = 'vsphere'
wdyt, change the constants or put .lower on the platform?